guides/release/in-depth-topics/autotracking-in-depth.md

```diff
@@ -321,4 +321,84 @@ class ShoppingList {
 }
 ```
 
+## Caching of tracked properties
+
+In contrast with computed properties from pre-Octane, autotracked properties
+are not cached. This means that an autotracked property will always be
+recomputed, regardless of whether its dependencies have changed. The following
+example shows how this works:
+
+```js
+import { tracked } from '@glimmer/tracking';
+
+let count = 0;
+
+class Person {
+  @tracked firstName = 'Jen';
+  @tracked lastName = 'Weber';
+
+  get fullName() {
+    ++count;
+    return `${this.firstName} ${this.lastName}`;
+  }
+}
+
+let person = new Person();
+
+console.log(person.fullName); // Jen Weber
+console.log(count); // 1;
+console.log(person.fullName); // Jen Weber
+console.log(count); // 2;
+
+person.firstName = 'Jennifer';
+
+console.log(person.fullName); // Jennifer Weber
+console.log(count); // 3;
```

Comment:
To help address https://github.com/ember-learn/guides-source/issues/1480, let's not use a `Person` class. Can we use the following example instead?

```suggestion
class Photo {
  @tracked width = 600;
  @tracked height = 400;

  get aspectRatio() {
    count++;
    return this.width / this.height;
  }
}

let photo = new Photo();

console.log(photo.aspectRatio); // 1.5
console.log(count); // 1;
console.log(photo.aspectRatio); // 1.5
console.log(count); // 2;

photo.width = 800;

console.log(photo.aspectRatio); // 2
console.log(count); // 3;
```

Reply:
Cool, agree.